### ISSUE TITLE
proxy errors, no connectivity, windows

### ISSUE THREAD

**Issue by brandondees** [HandyMiner/HandyBrowser#11]
after installing on windows 10 i'm seeing messages like the following for any navigation attempt:

> unable to proxy proofofconcept: read udp 127.0.0.1:60005->127.0.0.1:53: i/o timeout

same behavior before and after the hsd syncs up. for hns domains i get a green icon and can view the blockchain dns records, but no pages are loading.

nextdns hns resolution is also not working for me on the same machine, so i'm not sure if this is handybrowser specific or not, but the message seems to imply that something's wrong with the docker container's networking config. i'm not sure how to proceed troubleshooting wise, so here's an issue.

**Comment by smcki012** [comment]
Can you ensure NextDNS is disabled, and click the "Nuke Docker" button in the settings, which will reset your Docker image. If it still doesn't work, and NextDNS doesn't still for other browsers, may be a system issue that'd be hard for us to isolate.

**Comment by brandondees** [comment]
yep, i went another step further and manually removed the docker image as well. after a full rebuild with nextdns app disabled, things seem to be working again. nextdns resolution still isn't, but i presume that's a separate problem. I can use the handybrowser proxy for now, which is pretty cool.

**Comment by brandondees** [comment]
nevermind, still at least partially broken 

> unable to proxy parking.sinpapeles: read udp 127.0.0.1:52983->127.0.0.1:53: i/o timeout

seems to be working for some names and not others, perhaps it's related to incomplete DANE config or similar?

**Comment by smcki012** [comment]
Edit: Make sure your node is fully sync'd. It can't resolve a name that doesn't exist at its current block height. It'll use HNSD initially to bootstrap resolving names immediately on first install from active fullnodes on the network. Just be mindful of the first full sync, afterwards you'll be all good just open your browser (aka your node) regularly to sync. 

But browser uses solely the light client now, and HSD for initial bootstrapping/name resolving with the DANE proxy. 

If you find more issues still after you're fully sync'd and browser has restarted let us know details. And ensure the sites you test work elsewhere too.

**Comment by brandondees** [comment]
yeah this is after fully synced, and with sites that work on my other machine using nextdns

**Comment by smcki012** [comment]
Can you list any other sites this happens with? May be how the HNSD is interacting with certain sites.
